save-orchestrator/build.gradle.kts

```diff
@@ -14,16 +13,6 @@ tasks.withType<KotlinCompile> {
     }
 }
 
-tasks.withType<Test> {
-    retry {
```

Comment:
why?

Reply:
Supposed, that we won't have any flaky tests after this PR in orchestrator?

Reply:
yes. and more: `save-orchestrator` and `save-sandbox` don't have tests, they were moved to `save-orchestrator-common`.
Also, it looks like that retry on gradle level recreate everything instead of trying a failed test. I guess better adding retry on Junit level